pydantic/main.py

```diff
@@ -14,13 +14,16 @@
     Callable,
     Dict,
     List,
+    Literal,
```

Comment:
You should import it from `pydantic.typing`

Reply:
I'd started looking at the commit you had pointed to me and after a while realised the existing function `is_literal_type` could help out.

I'll investigate why this fails on Ubuntu 3.6 and 3.7 Python and not Windows / Mac. I'd written this on Ubuntu (testing with 3.6, 3.7 and 3.8) so this is a surprise.